server/liveblog/themes/themes_assets/default/theme.json

```diff
@@ -462,4 +462,4 @@
           "credit:": "credit:"
         }
     }
-}
+}
```

Comment:
please remove this from the commit, not relevant.

Reply:
It is a commit from devel branch which came to feature branch by its reference. and its very old commit to devel so, for now, I do not think it makes sense to do this. but for future, I will keep in mind for sure